graphics/pympress/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        cimbali pympress 1.8.5 v
+revision            0
+
+categories          graphics python
+supported_archs     noarch
+platforms           {darwin any}
+license             GPL-2
+maintainers         {@flyingsamson tuxcad.de:flyingsamson} openmaintainer
+description         PDF presentation tool
+long_description    ${name} is a {*}${description} designed for dual-screen \
+                    setups such as presentations and public talks
+
+checksums           rmd160  72fe51f878c139ef7f6306628445e6a9bf114817 \
+                    sha256  d8c155690b271567095b7e39265a8491448db7b2d1dce25bab8f34bdacaa69cc \
+                    size    260319
+
+python.default_version \
+                    312
+
+depends_build-append \
+                    port:py${python.version}-babel
+
+depends_run-append  port:cairo \
+                    port:gdk-pixbuf2 \
+                    port:gobject-introspection \
```

Comment:
please use `path:lib/pkgconfig/gobject-introspection-1.0.pc:gobject-introspection` so that the `gobject-introspection-devel` port could also satisfy this dependency

Reply:
Done